static/html/messageview.html

```diff
@@ -64,7 +64,9 @@ <h1><a href="/">Technochat</a></h1>
                 <p>Link: </p>
                 <div id="result_link"></div>
```

Comment:
`class="result__link"`?

Reply:
same for preview, result_text
also should change "Check your message here" to "Your message" as we have at the main page